src/common/mmap.h

```diff
@@ -85,6 +96,14 @@ void *util_map_tmpfile(const char *dir, size_t size, size_t req_align);
 #define RANGE_RW(addr, len, is_dev_dax) RANGE(addr, len, is_dev_dax, rw)
 #define RANGE_NONE(addr, len, is_dev_dax) RANGE(addr, len, is_dev_dax, none)
 
+/* pmem mapping type */
+enum pmem_map_type {
+	PMEM_DEV_DAX,	/* device dax */
+	PMEM_MAP_SYNC,	/* mapping with MAP_SYNC flag on dax fs */
+
+	MAX_PMEM_TYPE
```

Comment:
Please add comment here or change the name. Why MAX_*?

Reply:
there are two reasons:
1.having a last element of an enum be a MAX_ of something allows you to declare an array with enough space to fit all elements.
2. Every "limit" we declare internally is prefixed with MAX_ so that whenever we need to create an array, iterate over all enum types etc, we can just type MAX_ and let the automatic code completion show us all available options.

Reply:
👍